scripts/scaling/extrapolate_n.py

```diff
@@ -0,0 +1,104 @@
+import argparse
+import os
+from typing import Dict
+
+import matplotlib.pyplot as plt
+
+from olmo.aliases import PathOrStr
+from olmo.scaling.scaling_laws.curve_fit import (
+    chinchilla_fit,
+)
+from olmo.scaling.scaling_laws.extrapolate_n import (
+    CurveFitConfig,
+    get_data_at_d,
+    plot_n_scaling_at_d,
+)
+from olmo.scaling.scaling_laws.utils import validation
+
+VAL_KEYS = [f'eval/{val}/CrossEntropyLoss' for val in validation]
+
+CONFIG_BY_N = {
+    21266432: {
```

Comment:
We should have more human-readable names here

Reply:
`CONFIG_BY_MODEL_SIZE`?

Reply:
I mean, the keys. They can be "20M", "1B" etc. ?

Reply:
Thanks! I have taken care of this.